### PR TITLE
[Maybe WIP] Moved user related functions to user model

### PR DESCRIPTION
**Pull request by Ommy**
Moved away most of the user details logic to the User model.
Sessions holds a global user object that gets updated on login.

### CODE REVIEW

**Comment by Trinovantes** [comment]
better cohesion this time

+1

**Comment by Kapin** [comment]
Can we merge #371 first?

File: src-backbone/app/js/views/common/loggedInRightNavbarView.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 const LanguageSelectorView  = require('./languageSelectorView');
 const User                  = require('models/user');
-const USER_STORAGE_KEY      = require('models/session').USER_STORAGE_KEY;
 
 
 module.exports = Marionette.LayoutView.extend({
 
 	initialize: function() {
-        this.model = new User(App().session.get(USER_STORAGE_KEY));
+        this.model = App().session.user;
```
Comment (Kapin) [inline src-backbone/app/js/views/common/loggedInRightNavbarView.js]: indenting

**Comment by Kapin** [comment]
1 nit otherwise lgtm

File: src-backbone/app/js/views/common/loggedInRightNavbarView.js (hunk 1 above)
Comment (Ommy) [inline src-backbone/app/js/views/common/loggedInRightNavbarView.js]: done

**Pull request merged**